deploy/ansible/roles/iroha-docker/defaults/main.yml

```diff
@@ -21,29 +21,51 @@ custom_hostnames: false
 # deployments for testing purposes.
 #
 # Default: false
-overlay_network: false
+iroha_overlay_network: false
 
 ## Deployment configs
-container_basename: iroha
+iroha_container_basename: iroha
 # Path on a local machine for generated configs prior moving them on the remote
-config_local_dir: /tmp/iroha-ansible-conf
+iroha_config_local_dir: /tmp/iroha-ansible-conf
 # Path on a remote machine for generated configs
-deploy_dir: /opt/iroha-deploy
+iroha_deploy_dir: /opt/iroha-deploy
 # Overlay network name
 # If using overlay network plugin (like Calico) the network must be created prior running this role
-# The network will be created automatically if deploying locally or with `overlay_network`
+# The network will be created automatically if deploying locally or with `iroha_overlay_network`
 # set to `false`
 iroha_network_name: iroha-net
 
 # The role is incompatible with Iroha versions below RC2 since keys format has changed.
 # Apply the patch (files/old-keys-format.patch) if you need support for previous Iroha versions.
-iroha_docker_tag: 1.0.0_rc2
-postgres_docker_tag: '9.5'
+iroha_docker_tag: 1.0.0_rc5
+iroha_postgres_docker_tag: '9.5'
```

Comment:
Can we bump version to postures 10?

Reply:
The minimal supported version of Postgres is 9.5, AFAIK. It is used in CI to verify compatibility of each build. By using v9.5 we are confident that no breaking changes exist in later versions (though, they probably also play nice with Iroha. But we cannot say for sure).